addons/mail/static/src/core/web/follower.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.Follower">
+        <DropdownItem class="'o-mail-Follower d-flex justify-content-between p-0'">
+            <a class="o-mail-Follower-details d-flex align-items-center flex-grow-1 px-3 o-min-width-0" t-att-class="{ 'o-inactive fst-italic opacity-25': !props.follower.is_active }" href="#" role="menuitem" t-on-click.prevent="(ev) => this.onClickDetails(ev)">
+                <img class="o-mail-Follower-avatar me-2 rounded o_object_fit_cover" t-att-src="props.follower.partner.avatarUrl" alt=""/>
+                <span class="flex-shrink text-truncate" t-esc="props.follower.partner.name"/>
+            </a>
+            <t t-if="props.follower.isEditable">
+                <button class="o-mail-Follower-action btn btn-icon rounded-0" title="Edit subscription" aria-label="Edit subscription" t-on-click="(ev) => this.onClickEdit(ev)">
+                    <i class="fa fa-pencil"/>
+                </button>
+                <button class="o-mail-Follower-action btn btn-icon rounded-0" title="Remove this follower" aria-label="Remove this follower" t-on-click="(ev) => this.onClickRemove(ev)">
+                    <i class="fa fa-remove"/>
+                </button>
```

Comment:
- `fa-fw` on icon too.
- `title` is read by screen-readers, so no need to define `aria-label` when the value is the same.